server/apps/archive/archive.py

```diff
@@ -143,6 +143,15 @@ class ArchiveService(BaseService):
     packageService = PackageService()
     mediaService = ArchiveMediaService()
 
+    def on_fetched(self, docs):
+        """
+        Overriding this to handle existing data in Mongo & Elastic
+        """
+
+        for item in docs[config.ITEMS]:
+            if 'pubstatus' in item:
+                item['pubstatus'] = item.get('pubstatus', 'Usable').capitalize()
```

Comment:
:+1: 
Can we change to schema to have pubstatus as lowercase (http://cv.iptc.org/newscodes/pubstatusg2/usable)? 
We can easily handle string manipulation on output as well as css.